homeassistant/components/tomorrowio/strings.json

```diff
@@ -26,7 +26,8 @@
         "title": "Update Tomorrow.io Options",
         "description": "If you choose to enable the `nowcast` forecast entity, you can configure the number of minutes between each forecast. The number of forecasts provided depends on the number of minutes chosen between forecasts.",
         "data": {
-          "timestep": "Min. Between NowCast Forecasts"
+          "timestep": "Min. Between NowCast Forecasts",
+          "max_api_requests": "Maximum number of allowed API calls per day"
```

Comment:
`"api_calls_per_day": "Number of API calls per day"`

Reply:
Well, they *aren't* the number of API calls per day. They're the upper ceiling - there's some calculation happening when using multiple tomorrow.io integrations. So I think `max_api_calls_per_day` is more accurate.

Reply:
It's really a target and not a maximum. If HA gets restarted a few times during the day, then the number might exceed this value.

But at any rate, just make the naming consistent everywhere.

Reply:
> It's really a target and not a maximum. If HA gets restarted a few times during the day, then the number might exceed this value.
> 
> But at any rate, just make the naming consistent everywhere.

Planning to. To be honest I only made the PR to get an opinion on how to implement it - it's far, far, far from ready. 😭